docs/DeveloperGuide.md

```diff
@@ -299,26 +274,39 @@ the filtered transaction with a success message.
     - 2a1. UniCa$h displays an error message, requests for command to be re-entered.
     - Use case resumes at step 1
 
-**Use Case: UC08 - Clear All Transactions**
+
+**Use Case: UC06 - Clear all transactions**
 
 **MSS:**
 1. User enters the command to clear all transactions with the correct format. (i.e. no parameters)
 2. User submits the request.
 3. UniCa$h deletes all transactions in the transactions list.
 4. UniCa$h displays success message.
 
-   Use Case ends
+   Use Case ends.
 
 **Extensions**
 - 2a. User enters an incorrect format.
   - 2a1. UniCa$h displays an error message with the correct command format.
   - Use case resumes at step 1.
 
-- 3a. UniCa$h finds an empty transactions list.
-  - 3a1. UniCa$h displays a message saying that transactions list is empty.
-  - Use Case resumes at step 1
+**Use Case: UC07 - Reset UniCa$h**
+
+**MSS:**
+1. User enters the command to reset UniCa$h with the correct format. (i.e. no parameters)
```

Comment:
I believe @woojiahao mentioned in our Telegram group that we should avoid saying the user enters the command in the "correct format" under the MSS because then in the extension, it seems a little off to suddenly say that the user's input is in an incorrect format. Perhaps you could just leave remove the "with the correct format" part in this line

Reply:
Oh right that makes sense, thanks!

Reply:
I've gone ahead and fixed the others as well